MetaMask/metamask-extension
# Show a balance for the Token To, update position of the MM fee, removes a link

## Explanation
- Shows balance for the Token To
- Updates position of the MM fee and adds a link to view all quotes
- Removes a link from the Quote rate

## Screenshots
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/9f3ddc84-34e3-4c21-903a-91356e00d64a)

## Testing
- On the homepage click on the "Swap" button
- Select Token From, Token To where you have some balance and amount
- See a balance for Token To under its symbol, no link in the Quote rate line and a new line with the MM fee info + link to view all quotes
